cmu-sei/juneberry
# Bugfix/install data

Using `pip install -e .` "works" because it has a link the entire directory including schemas.  For a real `pip install .` we need to also tell it to install data files such as the schemas.

To test:

Inside the container do `pip install -e .` See that `jb_train tabular_multiclass_sample` runs.

Now do `pip install .`. See that `jb_train tabular_multiclass_sample` still runs. :)
